meta/main.yml

```diff
@@ -4,11 +4,14 @@ galaxy_info:
   description: Install upstream Nginx
   company: Open Microscopy Environment
   license: BSD
-  min_ansible_version: 2.1
+  min_ansible_version: 2.6
```

Comment:
Is the bump here related to a particular feature or the fact 2.6 is our current production Ansible version?

Reply:
We haven't tested 2.1 for ages and this contains some big changes.

Reply:
`import_tasks` requires 2.4 https://docs.ansible.com/ansible/2.4/import_tasks_module.html